sdk/core/azure-core-v2/src/main/java/com/azure/v2/core/implementation/AccessTokenCache.java

```diff
@@ -21,6 +21,7 @@
 
 /**
  * A token cache that supports caching a token and refreshing it.
+ * TODO: @g2vinay, make the implementation in client-core public API and remove it from this layer.
```

Comment:
Please do this.

Reply:
Now, we'll make it public API in azure-core-v2